### ISSUE TITLE
Requesting HaTeX version bound bumps

### ISSUE THREAD

**Issue by ysangkok** [haskell-infra/hackage-trustees#331]
In December, I made a PR for HaTeX about bumping the bounds of its Bytestring dependency, and it was merged. Afterwards, I asked for a revision, but I haven't gotten a reply:

* https://github.com/Daniel-Diaz/HaTeX/pull/153#issuecomment-996862209

Mid-January, I realized that also Hashable and Text should be bumped, which I also made a PR for, but I haven't gotten a reply:

* https://github.com/Daniel-Diaz/HaTeX/pull/154

Would it be possible for a trustee to bump these three bounds of the latest Hackage version, which is 3.22.3.0?

This is how I verified:

```
$ cabal get HaTeX
Unpacking to HaTeX-3.22.3.0/
$ cd HaTeX-3.22.3.0 
$ # This is where I manually edit the cabal file to upgrade the major PVP versions of each of these packages
$ cabal test --constraint='text>=2' --constraint='hashable>=1.4' --constraint='bytestring>=0.11'
Test suite parser-tests: PASS
1 of 1 test suites (1 of 1 test cases) passed.
Test suite hatex-test: PASS
1 of 1 test suites (1 of 1 test cases) passed.
```

Or if it is too early, please let me know. I deliberately waited longer than for #325. It sounds like I should wait more time if the maintainer is not active on Github. But @Daniel-Diaz is active, so that is why I only waited 24 days after the last PR.

**Comment by andreasabel** [comment]
@ysangkok: Seems like such an operation is within the trustee policy: https://github.com/haskell-infra/hackage-trustees/blob/master/policy.md#2-metadata-only-changes-relaxing-constraints

My only reservation is that the testsuite of HaTeX is just a stub and covers next to nothing.

Would you please contact the author @Daniel-Diaz by email (given under his profile), maybe that helps to get his attention?

If that does not help, we could risk to go ahead here, since the latest `bytestring`, `text`, and `hashable` bumps are not known for causing problems.

**Comment by ysangkok** [comment]
Yes, I have sent another email just now! Let's wait.

**Comment by ysangkok** [comment]
The issue was resolved with version 3.22.3.1 of HaTeX! Sorry for the disturbance.
